manage_breast_screening/participants/models.py

```diff
@@ -210,6 +210,9 @@ def previous(self) -> "ScreeningEpisode | None":
         except IndexError:
             return None
 
+    def __str__(self):
+        return f"{self.participant.full_name} - {self.protocol} {self.created_at.strftime('%Y-%m-%d %H:%M:%S') if self.created_at else ''}"
```

Comment:
what's the reason for including created_at in the string representation? Just thinking if we do it here we will probably want it for all the models

Reply:
We can also accomplish that with extra columns - e.g. for appointments I configured it like this

```
    list_display = [
        "name",
        "clinic_slot__starts_at",
        "clinic_slot__duration_in_minutes",
        "statuses__state",
    ]
```

This is also reminding me that we need some timestamps for when a screening episode was opened and closed. I'm not sure we should reuse `created_at` for application logic.